### ISSUE TITLE
Sympa Accessibility Issue - 17 - Screen reader announces the text fields as “edit blank”.

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1748]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
Using a Screen Reader, there needs to be a clear Labels required to understand the purpose of the text field so the screen reader doesn't announce nothing.

Actual behavior
---------------
Screen reader announces the text fields as “edit blank”.

Steps to reproduce
------------------
Using a screen reader hover over any text field.

Screen reader announces the text fields as “edit blank”. Labels are required to understand the purpose of the text field.
[WCAG Level A 3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)

![image](https://github.com/sympa-community/sympa/assets/144282683/f3f9556b-585e-45bc-8e64-0e32690dac84)

Fig 18 Focus order issue.

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023 by the University of Illinois at Urbana - Champaign, Technology Services Accessibility test team

[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13504926/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)
